### ISSUE TITLE
Chord/articulation interaction

### ISSUE THREAD

**Issue by craigsapp** [rism-ch/verovio#494]
The following example shows a funny behavior in verovio, where the articulations are placed on the wrong side of chord noteheads in certain cases (measures 2 and 3):

<img width="453" alt="screen shot 2017-02-28 at 9 26 30 am" src="https://cloud.githubusercontent.com/assets/3487289/23416601/54c84182-fd98-11e6-8acd-c55e4fc12801.png">

It seems that the articulation is placed on the chord before the stem direction of the chord is finally determined. Probably the articulation placement algorithm is using the top note of the chord when determining the stem direction, but later when the stem is actually applied, all chord notes are considered.   I tried using `chord@artic` and the same rendering occurred (so not related to artic subelements of chord specifically).

I don't recall seeing this before, so it could be related to newer code additions.

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2017-02-28T09:26:34" version="0.9.14-dev-fab327f">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000000304755543">
                        <staffGrp xml:id="staffgrp-0000001056958380">
                            <staffDef xml:id="staffdef-0000001233079630" clef.shape="G" clef.line="2" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000001348915382">
                        <measure xml:id="measure-L3" n="1">
                            <staff xml:id="staff-L3F1N1" n="1">
                                <layer xml:id="layer-L3F1N1" n="1">
                                    <beam xml:id="beam-L4F1-L5F1">
                                        <note xml:id="note-L4F1" dur="8" oct="4" pname="b">
                                            <accid xml:id="accid-L4F1" accid.ges="n" />
                                            <artic xml:id="artic-L4F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L5F1" dur="8" oct="4" pname="a">
                                            <accid xml:id="accid-L5F1" accid.ges="n" />
                                            <artic xml:id="artic-L5F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L6F1-L7F1">
                                        <chord xml:id="chord-L6F1" dur="8">
                                            <note xml:id="note-L6F1S1" oct="4" pname="b">
                                                <accid xml:id="accid-L6F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L6F1S2" oct="5" pname="d">
                                                <accid xml:id="accid-L6F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L6F1" artic="stacc" />
                                        </chord>
                                        <chord xml:id="chord-L7F1" dur="8">
                                            <note xml:id="note-L7F1S1" oct="4" pname="a">
                                                <accid xml:id="accid-L7F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L7F1S2" oct="5" pname="c">
                                                <accid xml:id="accid-L7F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L7F1" artic="stacc" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L8">
                            <staff xml:id="staff-L8F1N1" n="1">
                                <layer xml:id="layer-L8F1N1" n="1">
                                    <beam xml:id="beam-L9F1-L10F1">
                                        <note xml:id="note-L9F1" dur="8" oct="4" pname="a">
                                            <accid xml:id="accid-L9F1" accid.ges="n" />
                                            <artic xml:id="artic-L9F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L10F1" dur="8" oct="4" pname="g">
                                            <accid xml:id="accid-L10F1" accid.ges="n" />
                                            <artic xml:id="artic-L10F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L11F1-L12F1">
                                        <chord xml:id="chord-L11F1" dur="8">
                                            <note xml:id="note-L11F1S1" oct="4" pname="a">
                                                <accid xml:id="accid-L11F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L11F1S2" oct="5" pname="c">
                                                <accid xml:id="accid-L11F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L11F1" artic="stacc" />
                                        </chord>
                                        <chord xml:id="chord-L12F1" dur="8">
                                            <note xml:id="note-L12F1S1" oct="4" pname="g">
                                                <accid xml:id="accid-L12F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L12F1S2" oct="4" pname="b">
                                                <accid xml:id="accid-L12F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L12F1" artic="stacc" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L13">
                            <staff xml:id="staff-L13F1N1" n="1">
                                <layer xml:id="layer-L13F1N1" n="1">
                                    <beam xml:id="beam-L14F1-L15F1">
                                        <note xml:id="note-L14F1" dur="8" oct="4" pname="g">
                                            <accid xml:id="accid-L14F1" accid.ges="n" />
                                            <artic xml:id="artic-L14F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L15F1" dur="8" oct="4" pname="f">
                                            <accid xml:id="accid-L15F1" accid.ges="n" />
                                            <artic xml:id="artic-L15F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L16F1-L17F1">
                                        <chord xml:id="chord-L16F1" dur="8">
                                            <note xml:id="note-L16F1S1" oct="4" pname="g">
                                                <accid xml:id="accid-L16F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L16F1S2" oct="4" pname="b">
                                                <accid xml:id="accid-L16F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L16F1" artic="stacc" />
                                        </chord>
                                        <chord xml:id="chord-L17F1" dur="8">
                                            <note xml:id="note-L17F1S1" oct="4" pname="f">
                                                <accid xml:id="accid-L17F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L17F1S2" oct="4" pname="a">
                                                <accid xml:id="accid-L17F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L17F1" artic="stacc" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L18" right="end">
                            <staff xml:id="staff-L18F1N1" n="1">
                                <layer xml:id="layer-L18F1N1" n="1">
                                    <beam xml:id="beam-L19F1-L20F1">
                                        <note xml:id="note-L19F1" dur="8" oct="4" pname="f">
                                            <accid xml:id="accid-L19F1" accid.ges="n" />
                                            <artic xml:id="artic-L19F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L20F1" dur="8" oct="4" pname="e">
                                            <accid xml:id="accid-L20F1" accid.ges="n" />
                                            <artic xml:id="artic-L20F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L21F1-L22F1">
                                        <chord xml:id="chord-L21F1" dur="8">
                                            <note xml:id="note-L21F1S1" oct="4" pname="f">
                                                <accid xml:id="accid-L21F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L21F1S2" oct="4" pname="a">
                                                <accid xml:id="accid-L21F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L21F1" artic="stacc" />
                                        </chord>
                                        <chord xml:id="chord-L22F1" dur="8">
                                            <note xml:id="note-L22F1S1" oct="4" pname="e">
                                                <accid xml:id="accid-L22F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L22F1S2" oct="4" pname="g">
                                                <accid xml:id="accid-L22F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L22F1" artic="stacc" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
The image is not showing up for me

**Comment by ahwitz** [comment]
https://techcrunch.com/2017/02/28/amazon-aws-s3-outage-is-breaking-things-for-a-lot-of-websites-and-apps/ - I think this is responsible for Github right now as well, though I haven't heard official confirmation of that.

**Comment by craigsapp** [comment]
> The image is not showing up for me

I can see it...  It seems to be a temporary problem with github's aws service as @ahwitz mentions above.  I have been trying to submit another issue ;-), but I can't upload images at the moment.

**Comment by lpugin** [comment]
I suspect this related to the beam stem direction issue https://github.com/rism-ch/verovio/issues/414

**Comment by craigsapp** [comment]
Yes, that sound good.  The problem does not occur on notes which are not beamed:

<img width="628" alt="screen shot 2017-02-28 at 11 52 50 pm" src="https://cloud.githubusercontent.com/assets/3487289/23450638/1d2b8544-fe11-11e6-8372-6bdade338d09.png">

But the choice of above the noteheads is not clear (I reversed the order of the chords in the last measure and they still went above the noteheads, so it is not based on the first note/chord in a beam group)

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2017-02-28T23:51:59" version="0.9.14-dev-fab327f">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000001324444293">
                        <staffGrp xml:id="staffgrp-0000000958172824">
                            <staffDef xml:id="staffdef-0000001731566291" clef.shape="G" clef.line="2" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000000742531816">
                        <measure xml:id="measure-L2" n="1">
                            <staff xml:id="staff-L2F1N1" n="1">
                                <layer xml:id="layer-L2F1N1" n="1">
                                    <beam xml:id="beam-L3F1-L4F1">
                                        <note xml:id="note-L3F1" dur="8" oct="4" pname="a">
                                            <accid xml:id="accid-L3F1" accid.ges="n" />
                                            <artic xml:id="artic-L3F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L4F1" dur="8" oct="4" pname="g">
                                            <accid xml:id="accid-L4F1" accid.ges="n" />
                                            <artic xml:id="artic-L4F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L5F1-L6F1">
                                        <chord xml:id="chord-L5F1" dur="8">
                                            <note xml:id="note-L5F1S1" oct="4" pname="a">
                                                <accid xml:id="accid-L5F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L5F1S2" oct="5" pname="c">
                                                <accid xml:id="accid-L5F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L5F1" artic="stacc" />
                                        </chord>
                                        <chord xml:id="chord-L6F1" dur="8">
                                            <note xml:id="note-L6F1S1" oct="4" pname="g">
                                                <accid xml:id="accid-L6F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L6F1S2" oct="4" pname="b">
                                                <accid xml:id="accid-L6F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L6F1" artic="stacc" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L7">
                            <staff xml:id="staff-L7F1N1" n="1">
                                <layer xml:id="layer-L7F1N1" n="1">
                                    <beam xml:id="beam-L8F1-L9F1">
                                        <note xml:id="note-L8F1" dur="8" oct="4" pname="a">
                                            <accid xml:id="accid-L8F1" accid.ges="n" />
                                            <artic xml:id="artic-L8F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L9F1" dur="8" oct="4" pname="g">
                                            <accid xml:id="accid-L9F1" accid.ges="n" />
                                            <artic xml:id="artic-L9F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <chord xml:id="chord-L10F1" dur="8">
                                        <note xml:id="note-L10F1S1" oct="4" pname="a">
                                            <accid xml:id="accid-L10F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L10F1S2" oct="5" pname="c">
                                            <accid xml:id="accid-L10F1S2" accid.ges="n" />
                                        </note>
                                        <artic xml:id="artic-L10F1" artic="stacc" />
                                    </chord>
                                    <chord xml:id="chord-L11F1" dur="8">
                                        <note xml:id="note-L11F1S1" oct="4" pname="g">
                                            <accid xml:id="accid-L11F1S1" accid.ges="n" />
                                        </note>
                                        <note xml:id="note-L11F1S2" oct="4" pname="b">
                                            <accid xml:id="accid-L11F1S2" accid.ges="n" />
                                        </note>
                                        <artic xml:id="artic-L11F1" artic="stacc" />
                                    </chord>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L12" right="end">
                            <staff xml:id="staff-L12F1N1" n="1">
                                <layer xml:id="layer-L12F1N1" n="1">
                                    <beam xml:id="beam-L13F1-L14F1">
                                        <note xml:id="note-L13F1" dur="8" oct="4" pname="a">
                                            <accid xml:id="accid-L13F1" accid.ges="n" />
                                            <artic xml:id="artic-L13F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L14F1" dur="8" oct="4" pname="g">
                                            <accid xml:id="accid-L14F1" accid.ges="n" />
                                            <artic xml:id="artic-L14F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L15F1-L16F1">
                                        <chord xml:id="chord-L15F1" dur="8">
                                            <note xml:id="note-L15F1S1" oct="4" pname="g">
                                                <accid xml:id="accid-L15F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L15F1S2" oct="4" pname="b">
                                                <accid xml:id="accid-L15F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L15F1" artic="stacc" />
                                        </chord>
                                        <chord xml:id="chord-L16F1" dur="8">
                                            <note xml:id="note-L16F1S1" oct="4" pname="a">
                                                <accid xml:id="accid-L16F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L16F1S2" oct="5" pname="c">
                                                <accid xml:id="accid-L16F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L16F1" artic="stacc" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

And it also affects single notes in a beam group, but not if there are no chords in the beam group:

<img width="540" alt="screen shot 2017-02-28 at 11 59 06 pm" src="https://cloud.githubusercontent.com/assets/3487289/23450817/fabba9e8-fe11-11e6-88d4-9613d84ed8bb.png">

MEI test data:

```xml
<?xml version="1.0" encoding="UTF-8"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://relaxng.org/ns/structure/1.0"?>
<?xml-model href="http://music-encoding.org/schema/3.0.0/mei-all.rng" type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
<mei xmlns="http://www.music-encoding.org/ns/mei" meiversion="3.0.0">
    <meiHead>
        <fileDesc>
            <titleStmt>
                <title />
            </titleStmt>
            <pubStmt />
        </fileDesc>
        <encodingDesc>
            <appInfo>
                <application isodate="2017-02-28T23:58:50" version="0.9.14-dev-fab327f">
                    <name>Verovio</name>
                    <p>Transcoded from Humdrum</p>
                </application>
            </appInfo>
        </encodingDesc>
        <workDesc>
            <work>
                <titleStmt>
                    <title />
                </titleStmt>
            </work>
        </workDesc>
        <extMeta>
            <frames xmlns:humxml="http://www.humdrum.org/ns/humxml">
                <metaFrame n="18" token="!!!RDF**kern: &gt; = above" xml:id="loc18">
                    <frameInfo>
                        <startTime float="6" />
                        <frameType>reference</frameType>
                        <referenceKey>RDF**kern</referenceKey>
                        <referenceValue>&gt; = above</referenceValue>
                    </frameInfo>
                </metaFrame>
                <metaFrame n="19" token="!!!RDF**kern: &lt; = below" xml:id="loc19">
                    <frameInfo>
                        <startTime float="6" />
                        <frameType>reference</frameType>
                        <referenceKey>RDF**kern</referenceKey>
                        <referenceValue>&lt; = below</referenceValue>
                    </frameInfo>
                </metaFrame>
            </frames>
        </extMeta>
    </meiHead>
    <music>
        <body>
            <mdiv>
                <score>
                    <scoreDef xml:id="scoredef-0000000681724480">
                        <staffGrp xml:id="staffgrp-0000001078588407">
                            <staffDef xml:id="staffdef-0000000625132548" clef.shape="G" clef.line="2" n="1" lines="5" />
                        </staffGrp>
                    </scoreDef>
                    <section xml:id="section-0000000875822739">
                        <measure xml:id="measure-L2" n="1">
                            <staff xml:id="staff-L2F1N1" n="1">
                                <layer xml:id="layer-L2F1N1" n="1">
                                    <beam xml:id="beam-L3F1-L4F1">
                                        <note xml:id="note-L3F1" dur="8" oct="4" pname="a">
                                            <accid xml:id="accid-L3F1" accid.ges="n" />
                                            <artic xml:id="artic-L3F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L4F1" dur="8" oct="4" pname="g">
                                            <accid xml:id="accid-L4F1" accid.ges="n" />
                                            <artic xml:id="artic-L4F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L5F1-L6F1">
                                        <chord xml:id="chord-L5F1" dur="8">
                                            <note xml:id="note-L5F1S1" oct="4" pname="a">
                                                <accid xml:id="accid-L5F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L5F1S2" oct="5" pname="c">
                                                <accid xml:id="accid-L5F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L5F1" artic="stacc" />
                                        </chord>
                                        <chord xml:id="chord-L6F1" dur="8">
                                            <note xml:id="note-L6F1S1" oct="4" pname="g">
                                                <accid xml:id="accid-L6F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L6F1S2" oct="4" pname="b">
                                                <accid xml:id="accid-L6F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L6F1" artic="stacc" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L7">
                            <staff xml:id="staff-L7F1N1" n="1">
                                <layer xml:id="layer-L7F1N1" n="1">
                                    <beam xml:id="beam-L8F1-L9F1">
                                        <note xml:id="note-L8F1" dur="8" oct="4" pname="a">
                                            <accid xml:id="accid-L8F1" accid.ges="n" />
                                            <artic xml:id="artic-L8F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L9F1" dur="8" oct="4" pname="g">
                                            <accid xml:id="accid-L9F1" accid.ges="n" />
                                            <artic xml:id="artic-L9F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L10F1-L11F1">
                                        <note xml:id="note-L10F1" dur="8" oct="5" pname="c">
                                            <accid xml:id="accid-L10F1" accid.ges="n" />
                                            <artic xml:id="artic-L10F1" artic="stacc" />
                                        </note>
                                        <chord xml:id="chord-L11F1" dur="8">
                                            <note xml:id="note-L11F1S1" oct="4" pname="g">
                                                <accid xml:id="accid-L11F1S1" accid.ges="n" />
                                            </note>
                                            <note xml:id="note-L11F1S2" oct="4" pname="b">
                                                <accid xml:id="accid-L11F1S2" accid.ges="n" />
                                            </note>
                                            <artic xml:id="artic-L11F1" artic="stacc" />
                                        </chord>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                        <measure xml:id="measure-L12" right="end">
                            <staff xml:id="staff-L12F1N1" n="1">
                                <layer xml:id="layer-L12F1N1" n="1">
                                    <beam xml:id="beam-L13F1-L14F1">
                                        <note xml:id="note-L13F1" dur="8" oct="4" pname="a">
                                            <accid xml:id="accid-L13F1" accid.ges="n" />
                                            <artic xml:id="artic-L13F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L14F1" dur="8" oct="4" pname="g">
                                            <accid xml:id="accid-L14F1" accid.ges="n" />
                                            <artic xml:id="artic-L14F1" artic="stacc" />
                                        </note>
                                    </beam>
                                    <beam xml:id="beam-L15F1-L16F1">
                                        <note xml:id="note-L15F1" dur="8" oct="5" pname="c" stem.dir="up">
                                            <accid xml:id="accid-L15F1" accid.ges="n" />
                                            <artic xml:id="artic-L15F1" artic="stacc" />
                                        </note>
                                        <note xml:id="note-L16F1" dur="8" oct="4" pname="b" stem.dir="up">
                                            <accid xml:id="accid-L16F1" accid.ges="n" />
                                            <artic xml:id="artic-L16F1" artic="stacc" />
                                        </note>
                                    </beam>
                                </layer>
                            </staff>
                        </measure>
                    </section>
                </score>
            </mdiv>
        </body>
    </music>
</mei>
```

**Comment by lpugin** [comment]
Duplicates with #414

**Comment by lpugin** [comment]
Fixed with 7de376debeaa69323358c03f053b1b24595ab0b2

**Comment by craigsapp** [comment]
Looks good:

<img width="752" alt="screen shot 2017-03-06 at 1 27 11 pm" src="https://cloud.githubusercontent.com/assets/3487289/23630683/d16db15e-0270-11e7-8cc5-e243c5a78327.png">
